### PR TITLE
add more info about RichText field and the usage of transforms and RichTextValue

### PR DESCRIPTION
**Pull request by MrTango**
part of #1445

### CODE REVIEW

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
The `RichText` field allows for alternative markups and content filtering.
The following code sample shows how to create a schema with a `RichText` field.
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]: Minor nit-pick, but if we Black-ify code, then `"` is preferred over `'`.

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
#### `RichTextValue`
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
The `RichText` field usually does not store a string.
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
:   The MIME type of the original markup, for example, `text/html` or `text/structured`.
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]: More nitpicks s/'/".

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
##### `RichText` fields in templates
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
If you need to write your own transforms, take a look at [Changing Portal Transforms Settings via Python](https://5.docs.plone.org/develop/plone/misc/portal_transforms.html).
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]: More nitpick.

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
The first traversal name segment gives the name of the field on the context (`body` in this case).
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
The second and third segments give the output MIME type.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
This is good stuff. Just a few minor nitpicks, and it passes my English and MyST syntax and grammar check. It should still get reviewed by someone with technical familiarity.

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
#### `NamedBlobImage`
```

File: docs/backend/fields.md
Comment (stevepiercy) [inline docs/backend/fields.md]:
```suggestion
The following example shows how to create an `Image` object, and attach the image file to it.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: docs/backend/fields.md
Comment (davisagli) [inline docs/backend/fields.md:119]: It's probably better to pass the image in the call to create(), so that the image_scales get indexed during the object creation. (Otherwise you would need a separate reindex after this)

**Review by davisagli — COMMENTED** [review]
Looks good from a technical perspective. One small suggestion

File: docs/backend/fields.md
Comment (stevepiercy) [reply docs/backend/fields.md:119]: @MrTango would you please respond to @davisagli's suggestion?

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I committed my suggestions. A couple of nitpicks for Blackifying the code and another suggestion from @davisagli need to be addressed. @MrTango let's finish this PR.

File: docs/backend/fields.md
Comment (MrTango) [reply docs/backend/fields.md:119]: @davisagli thx for the hint, better now?

**Review by stevepiercy — APPROVED** [review]
Linting changes look good. Let's wait for @davisagli's review before merging.

**Review by davisagli — APPROVED** [review]

**Pull request merged**
